test/base_test.py

```diff
@@ -207,32 +204,22 @@ def get__array_struct__(self):
 
     if pygame.HAVE_NEWBUF:
         from pygame.tests.test_utils import buftools
-        def test_newbuf(self):
-            self.NEWBUF_test_newbuf()
-        def test_PgDict_AsBuffer_PyBUF_flags(self):
-            self.NEWBUF_test_PgDict_AsBuffer_PyBUF_flags()
-        def test_PgObject_AsBuffer_PyBUF_flags(self):
-            self.NEWBUF_test_PgObject_AsBuffer_PyBUF_flags()
-        def test_bad_format(self):
-            self.NEWBUF_test_bad_format()
 
     def NEWBUF_assertSame(self, proxy, exp):
         buftools = self.buftools
         Importer = buftools.Importer
         self.assertEqual(proxy.length, exp.len)
         imp = Importer(proxy, buftools.PyBUF_RECORDS_RO)
-        try:
-            self.assertEqual(imp.readonly, exp.readonly)
-            self.assertEqual(imp.format, exp.format)
-            self.assertEqual(imp.itemsize, exp.itemsize)
-            self.assertEqual(imp.ndim, exp.ndim)
-            self.assertEqual(imp.shape, exp.shape)
-            self.assertEqual(imp.strides, exp.strides)
-            self.assertTrue(imp.suboffsets is None)
-        finally:
-            imp = None
```

Comment:
Similar to previous comment, this `imp = None` part is probably needed (I'm guessing).

Reply:
Same question as above: why would it be needed? Do you think, it could be related to the `AssertionError` crash above ? I'll try it out..

Reply:
no, my remark was non sens, it can not be related since the crash happens as of commit (917f364) where as the `del` are removed on my last commit :(